tests/fr/media_player_HassMediaUnpause.yaml

```diff
@@ -0,0 +1,10 @@
+language: fr
+tests:
+  - sentences:
+      - "reprends sur TV"
+      - "lecture sur TV"
```

Comment:
Now that I see these test. I think we're missing a sentence that would be 
`reprend/reprendre la lecture sur la tv`